docs/scalar_tensor.rst

```diff
@@ -3,6 +3,33 @@
 Scalar fields
 =============
 
+**Taichi fields** are used to store data.
+
+Field **elements** could be either scalar, vector or matrix (see :ref:`matrix`).
+We'll only talk about **scalar fields** in this chapter.
+
+Field **dimensions** could be either 0D, 1D, 2D, 3D, ...
+
+- 0D fields are simply scalars, it only contains one scalar value.
```

Comment:
```suggestion
- 0D fields are simply scalars, they only contain one scalar value on each entry.
```

Reply:
Sorry, didn't get the point... A 0D field only have one entry, why we need `each`?

Reply:
```suggestion
- 0D fields are simply scalars, they only contain one scalar value.
```

Reply:
Yes, I agree.